multiworld/core/image_env.py

```diff
@@ -117,7 +117,7 @@ def sample_goals(self, batch_size):
         return goals
 
     def compute_rewards(self, achieved_goals, desired_goals, info):
-        return - np.linalg.norm(achieved_goals - desired_goals, axis=1)
+        return - np.linalg.norm(achieved_goals - desired_goals, ord=self.norm_order, axis=1)
```

Comment:
Either (1) add `norm_order` parameter to the `__init__` or (2) don't reference variable. Remember, the `ImageEnv` is a _wrapper_ not a subclass, so `self.norm_order` doesn't exist unless we want to say "`ImageEnv`s can only wrap vectoriz-able environments", which I don't think we do.

In general, I think we should just undo this diff, i.e. not reference `self.norm_order`. If someone wants to just use a vectorize environment, then just don't wrap the env with a `ImageEnv`. I'm not sure if someone will ever want to have a vectorized `ImageEnv`, but if they do, they can implement that when needed.